website/src/_data/capability-matrix.yml

```diff
@@ -1367,3 +1367,102 @@ categories:
             l1: 'No'
             l2: pending model support
             l3: ''
+  - description: Additional features
+    anchor: misc
+    color-b: 'aaa'
+    color-y: 'bbb'
+    color-p: 'ccc'
+    color-n: 'ddd'
+    rows:
+      - name: Drain
+        values:
+          - class: model
+            l1: 'Partially'
+            l2: 
+            l3: APIs and semantics for draining a pipeline are under discussion. This would cause incomplete aggregations to be emitted regardless of trigger and tagged with metadata indicating it is incomplated.
+          - class: dataflow
+            l1: 'Partially'
+            l2: 
+            l3: Dataflow has a native drain operation, but it does not work in the presence of event time timer loops. Final implemention pending model support.
+          - class: flink
+            l1: 
+            l2: 
+            l3: 
```

Comment:
How about this?

'Flink supports taking a "savepoint" of the pipeline and shutting the pipeline down after its completion.'

I suppose drain is semantically different from "checkpoint, then stop pipeline" because a pipeline continues to process data until the checkpoint is complete.

Reply:
I think that the PaneInfo has a bit that says "is drain" and that it causes to emit unfinished aggregations. @reuvenlax would know a bit more. Since the feature is unfinished anyhow I am happy to go with this wording and improve later.